src/main/java/com/solvdLaba/airport/Passenger.java

```diff
@@ -10,32 +11,30 @@ public class Passenger implements Security{
     private boolean isTSAChecked;
     private Boarding boardingPass;
 
-    public Luggage getLuggage() {
-        return luggage;
-    }
-
-    public void setName(String name) {
+    public Passenger(String name, int age){
         this.name = name;
-    }
-
-    public void setAge(int age) throws InvalidAgeException {
         this.age = age;
-        if (age < 0){
-            throw new InvalidAgeException("Age cannot be less than zero", new IllegalArgumentException() {
-            });
-        }
-
+        MyLogger.info("Passenger getting created:  " + name);
     }
-
-    public void setLuggage(Luggage luggage) {
-        this.luggage = luggage;
+    public boolean isTSAChecked() {
+        return isTSAChecked;
     }
 
 
-    public Passenger(String name, int age){
-        this.name = name;
-        this.age = age;
-      //  MyLogger.info("Passenger getting created:  " + name);
+    protected boolean hasValidBoardingPass(Passenger p1) throws InvalidBoardingPassException {
+        if (p1.getBoardingPass() == null) {
+            throw new InvalidBoardingPassException("Boarding pass is missing", p1.getName());
+        }
+
+        return true;
+    }
+    @Override
+    public void skipSecurityCheckLine(boolean checked){
+        if (checked){
+            this.isTSAChecked = true;
+        }else{
+            this.isTSAChecked = false;
+        }
```

Comment:
```suggestion
        this.isTSAChecked = checked;
```

Reply:
Here you requested to initialize isTSAChecked. I thought the code in if-else condition is enough to do initialize. But anyway I did as you suggested. If it's possible could you please tell the reason of adding this line: this.isTSAChecked = checked; ? I just want to make sure did I get your request correctly. Thank you

Reply:
It is not adding the line, it is replacing the lines.
Your if-else statement basically results in the same line as i suggested. 
You are saying: if result is positive save positive into field, if result is negative -> save negative into field. 
I am saying: put the result into the field.